tn_api_v2_core.yml

```diff
@@ -0,0 +1,948 @@
+openapi: "3.0.0"
+
+info:
+    version: 2.0.0
+    title: Trip Ninja API Documentation
+    description: |
+        <h2>Introduction</h2>
+        Trip Ninja uses a REST API with requests and responses in JSON format. For development and testing, the pre-production servers
+        are to be used. Once ready for live use, access to the production servers will be provided. <br  />
+        <h2>Trip Ninja API URLs</h2>
+        
+        Pre-Production: https://preprodapi.tripninja.io
+        
+        For security, Trip Ninja’s servers are only accessible from whitelisted IPs. Please contact your account manager to whitelist IPs you
+        will be using.
+        
+        Before you start ensure that:<br  />
+            - IP addresses have been provided to be whitelisted for our servers.<br  />
+            - API username and password have been provided by Trip Ninja.<br  />
+        
+        <h2>Workflow</h2>
+        The following diagram shows the typical data flow of API calls to Trip Ninja for a typical OTA integration.
```

Comment:
Wondering about the working in the sentence, think it could be improved a bit.        

Maybe something like:

The following diagram shows the data flow of API calls to Trip Ninja for an OTA integration using our new core non-emulation architecture. 
(im not sure what exactly to call it lol)

Reply:
cool will address!